public/themes/pterodactyl/js/frontend/console.js

```diff
@@ -148,16 +150,20 @@ function pushToTerminal(string) {
     }
 
     if (TerminalQueue.length > 0) {
+        var scrolledDown = isTerminalScrolledDown();
+        
         for (var i = 0; i < CONSOLE_PUSH_COUNT && TerminalQueue.length > 0; i++) {
             pushToTerminal(TerminalQueue[0]);
 
-            if (! $scrollNotify.is(':visible')) {
-                window.scrollToBottom();
-            }
-
             window.ConsoleElements++;
             TerminalQueue.shift();
         }
+        
+        if (scrolledDown) {
```

Comment:
This is still inconsistent. It should be `if (isTerminalScrolledDown())` and then remove the `var scrolledDown` since it is unnecessary.

Reply:
As I said, once new lines are added, the console isn't scrolled down anymore. Therefore the check before adding them, and then scroll down after all are added.